internal/querycoordv2/task/scheduler.go

```diff
@@ -934,6 +956,11 @@ func (scheduler *taskScheduler) remove(task Task) {
 		scheduler.decExecutingTaskDelta(task)
 	}
 
+	if scheduler.tasks.Len() == 0 {
+		scheduler.segmentTaskDelta.printDetailInfos()
```

Comment:
Will this print be frequent?

Reply:
only task is empty and segment task delta cache is leak, will print the log